src/main/resources/static/templates/python-producer/src/producer.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+import random
+import string
+import sys
+from configparser import ConfigParser
+import time
+from confluent_kafka import Producer
+
+if __name__ == '__main__':
+    # Parse config from file `config.ini`
+    config_parser = ConfigParser()
+    config_parser.read('config.ini')
+    config = dict(config_parser['default'])
+
+    # creates a new producer instance
+    producer = Producer(config)
+
+    # produces message until terminated
+    try:
+        while True:
+            # produces a sample message
+            key = ''.join(random.choices(string.ascii_letters + string.digits, k=5))
+            value = ''.join(random.choices(string.ascii_letters + string.digits, k=10))
```

Comment:
Maybe we should produce random JSON values? Fine either way just thinking about what to start with.

Reply:
@derek1ee Mentioning you for visibility.